aws/client/client.go

```diff
@@ -89,7 +89,10 @@ const logReqMsg = `DEBUG: Request %s/%s Details:
 
```

Comment:
```go
const logDumpErrorMsg = `DEBUG ERROR: Request %s/%s Details:
---[ REQUEST DUMP ERROR ]----------------------
%s
-----------------------------------------------------`
```

Something of this nature, that way we have some consistency with the current logging.